Functions.Templates/Templates/IoTHubServiceBusQueueTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "defaultFunctionName": "IoTHubServiceBusQueueTriggerCSharp",
```

Comment:
defaultFunctionName should not have "CSharp" in it. We only support single language in a function app. This was added back when we allowed functions from multiple languages in the same app.